docs/integration/kubernetes.md

```diff
@@ -192,598 +192,593 @@ Container and Kubernetes metrics are collected along with labels and tags, which
-|etcd.proposals.pending|uint64|The current number of pending proposals to commit|
-|etcd.version|float64|Which version is running|
+|Metric Name|Unit| Description                             |
+|----|----|-----------------------------------------|
+|etcd.has_leader|uint64| whether or not a leader exists          |
+|etcd.leader.changes_seen|uint64| number of leader changes seen           |
+|etcd.proposals.applied|uint64| number of consensus proposals applied   |
+|etcd.proposals.committed|uint64| number of consensus proposals committed |
+|etcd.proposals.failed|uint64| number of failed proposals seen         |
+|etcd.proposals.pending|uint64| number of pending proposals to commit   |
+|etcd.version|float64| version of etcd running                 |
 
 #### Commit
-|Metric Name|Unit|Description|
-|----|----|-----------|
-|etcd.disk_backend_commit.p50latency|float64||
-|etcd.disk_backend_commit.p75latency|float64||
-|etcd.disk_backend_commit.p90latency|float64||
-|etcd.disk_backend_commit.p95latency|float64||
-|etcd.disk_backend_commit.p99latency|float64||
-|etcd.disk_backend_commit.total_num|uint64||
-|etcd.disk_backend_commit.duration|float64|The latency distributions of commit called by backend|
+|Metric Name|Unit| Description                       |
+|----|----|-----------------------------------|
+|etcd.disk_backend_commit.p50latency|float64| p50 latency of commit operations  |
+|etcd.disk_backend_commit.p75latency|float64| p75 latency of commit operations  |
+|etcd.disk_backend_commit.p90latency|float64| p90 latency of commit operations  |
+|etcd.disk_backend_commit.p95latency|float64| p95 latency of commit operations  |
+|etcd.disk_backend_commit.p99latency|float64| p99 latency of commit operations  |
+|etcd.disk_backend_commit.total_num|uint64| number of commit operations for etcd's disk bakend commits |
+|etcd.disk_backend_commit.duration|float64| duration of commit operations for etcd's disk backend in seconds |
 
 #### WAL
-|Metric Name|Unit|Description|
-|----|----|-----------|
-|etcd.disk_wal_fsync.p50latency|float64||
-|etcd.disk_wal_fsync.p75latency|float64||
-|etcd.disk_wal_fsync.p90latency|float64||
-|etcd.disk_wal_fsync.p95latency|float64||
-|etcd.disk_wal_fsync.p99latency|float64||
-|etcd.disk_wal_fsync.total_num|uint64||
-|etcd.disk_wal_fsync.duration|float64|The latency distributions of fsync called by WAL|
+|Metric Name|Unit| Description                      |
+|----|----|----------------------------------|
+|etcd.disk_wal_fsync.p50latency|float64| p50 latency of fsync operations  |
+|etcd.disk_wal_fsync.p75latency|float64| p75 latency of fsync operations  |
+|etcd.disk_wal_fsync.p90latency|float64| p90 latency of fsync operations  |
+|etcd.disk_wal_fsync.p95latency|float64| p95 latency of fsync operations  |
+|etcd.disk_wal_fsync.p99latency|float64| p99 latency of fsync operations  |
+|etcd.disk_wal_fsync.total_num|uint64| number of fsync (flush to disk) operations for etcd's write-ahead log (WAL) on the disk backend |
```

Comment:
Does "disk backend" make sense?  Why not just "disk"?

Reply:
The metric is named that way `disk backend`, for example we also have `disk_wal` in this case, if we name just disk can be confusing IMO